src/widget/helpbrowser.h

```diff
@@ -0,0 +1,20 @@
+#include <QPoint>
+#include <QTextBrowser>
+#include <QUrl>
+#include <QVariant>
+
+// forward declarations
+class QWidget;
+class QHelpEngine;
+
+class HelpBrowser : public QTextBrowser {
```

Comment:
What is the difference between HelpBrowser and HelpViewer? Please document all new classes.